target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -3,6 +3,10 @@
 PART_NAME=firmware
 REQUIRE_IMAGE_METADATA=1
 
+get_magic_long_vr2600v() {
```

Comment:
I actually noticed this. But if I remove my changes my branch will no longer update, as in my branch the old check is still in place. And merging around did break a lot.